### ISSUE TITLE
Pixel art texture is blurred.

### ISSUE THREAD

**Issue by cessar1** [Zylann/godot_heightmap_plugin#424]
Is it possible to use pixel art textures with this plugin, and if so, how can the issue of the texture not displaying correctly be addressed? Thanks in advance.

**Comment by Zylann** [comment]
Yes, but it's a bit more work than a checkbox: https://github.com/Zylann/godot_heightmap_plugin/issues/233#issuecomment-1745530875
